GherkinSyncTool/GherkinSyncTool/Synchronizers/TestRailSynchronizer/TestRailManager/Model/UpdateCaseRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace GherkinSyncTool.Synchronizers.TestRailSynchronizer.TestRailManager.Model
+{
+    public class UpdateCaseRequest
+    {
```

Comment:
What if we move the case from one folder to another? Will it be handled correctly?
Shouldn't we add SectionId property?

Reply:
Yep, we should handle this. Section logic will be implemented in a separate task.